django/santropolFeast/member/views.py

```diff
@@ -632,6 +632,34 @@ def get_context_data(self, **kwargs):
         return context
 
 
+class ClientStatusView(generic.DetailView):
+    model = Client
```

Comment:
Why not use `generic.ListView` https://docs.djangoproject.com/en/1.10/ref/class-based-views/generic-display/#listview
you are showing a page with the lists of `client_statuses`. 

> class django.views.generic.list.ListView¶
A page representing a list of objects.

> While this view is executing, self.object_list will contain the list of objects (usually, but not necessarily a queryset) that the view is operating upon.